src/bezier.jl

```diff
@@ -1,37 +1,39 @@
 using StableHashTraits
 
+const Point2d = Point2{Float64}
```

Comment:
This should be in Makie.jl (or maybe types.jl) for better visibility, imo. Maybe also with a 3d and 4d variant, since we'll likely need them if we switch to Float64 for internals in the future.

Reply:
It should actually go into GeometryBasics... Can still do that once we do the refactor, no?